mysite/fsm/models.py

```diff
@@ -82,78 +41,106 @@ def save_json_data(self, d=None, attr='data', doSave=True):
 )
 
 
-def load_json_id(name, pk):
+class JSONBlobMixin(object):
```

Comment:
We decided not to change code at this point and to make a note in issue tracker and possible using this function in future in general way.